MarkdownEditor.elm

```diff
@@ -0,0 +1,39 @@
+module MarkdownEditor where
+-- Elm’s "import" keyword works mostly like "require" in node.js.
```

Comment:
Are all the comments necessary? I think the module syntax is fairly straightforward after reading the initial elm tutorial.